### ISSUE TITLE
Tree: Screenreader does not read the state as "collapsed" when node gets collapsed

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52601]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
Issue type : Usability
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while screenreader is ON.
2. Navigate to Activity Bar and select "Explorer"(Cntrl+Shift+E) button.
3. Navigate to "File Explorer" section treeview items using "Tab".
4. Read all the treeview items at level 1, level2..and so on.
5. Expand/Collapse the treeview items using "Space". 

**Actual:**
NVDA does not read the "collapsed" state when any action is performed on the treeview items. 
It only reads state as "expanded" when any treeview item is expanded. It reads "collapsed" when we move to any other treeview item using up/down arrow keys.

**Expected:**
NVDA should read "collapsed" state instantaneously with each item when any action (collapse) is performed. It should not read the state when we navigate to other treeview items. 

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not get information whether the treeview item has been collapsed as action is performed on it. 

**Attachment For Reference:**
[A11y_VSCode_ViewExplorer_Usability_CollapsedStateNotAnnounced.pptx](https://github.com/Microsoft/vscode/files/2123504/A11y_VSCode_ViewExplorer_Usability_CollapsedStateNotAnnounced.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This is a general tree issues that the collapsed state is not read out after colapsing. Forwarding to @joaomoreno
